coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -8505,6 +8505,37 @@ def forward(self, x, y, z):
         )
 
 
+class TestAddmm(TorchBaseTest):
+    @pytest.mark.parametrize(
+        "compute_unit, backend, shapes, beta, alpha",
+        itertools.product(
+            compute_units,
+            backends,
+            ((2, 2, 2), (4, 5, 9)),
+            (1., 2.),
+            (1., 3.),
+        )
+    )
+    def test_addmm(self, compute_unit, backend, shapes, beta, alpha):
+
+        class TestModel(nn.Module):
+            def forward(self, x):
+                return torch.addmm(x, m1, m2, beta=beta, alpha=alpha)
+
+
+        m, n, p = shapes
+
+        # m1 @ m2 must be legal
+        m1 = torch.randn(m, n)
+        m2 = torch.randn(n, p)
+        # x must be the same shape as m1 @ m2
+        x_shape = (m, p)
+
+        self.run_compare_torch(
+            x_shape, TestModel(), backend=backend, compute_unit=compute_unit,
+        )
+
```

Comment:
Shall we also grab the converted model and assert the ops in it? E.g. when `linear` is applicable, we should get `linear`, otherwise we should get `matmul + add`

Reply:
I'm not sure I understand this comment. Are you assuming that optimization passes should convert `matul + add` to `linear` when possible? That doesn't really seem like functionality we should be testing in a op conversion unit test.